msal/changelog.txt

```diff
@@ -1,3 +1,17 @@
+2.4.0-preview
+=============
+- Cache fixes 
+   - Not remove account cache entities [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1365)
+   - Fix issue with sharing in memory cahce for different instances of the TokenCache [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1360)
+   - Clear in memory version of cache [MSAL issue #654](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
+   - Fix NullPointException when adalv3 account is used because there is no client info in adalv3 [MSAL Issue #653](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/654)
+   - RemoveAdalUser api now removes users by accountID and by displayableID to cover migration scenario from adalv3 to adalv4 to msalv2. [MSAL Issue #652](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/652)
+   - Fix for internal TokenCache.Clear api so ADAL persistent cache is also cleared [MSAL Issue #651](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/651)
+   - Optimize cache lookups with native iOS by modifying field for persistence on iOS. [PR](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1350)
+- Support [extended lifetime tokens](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1331)
+- Remove Chrome dependency for system browser on Android devices. See https://aka.ms/msal-net-system-browsers for more information. [MSAL issue #664](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/664)
+- Add support for b2clogin.com for b2c authorities [MSAL issue #669](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/669) [#632](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/632)
```

Comment:
>Add support for b2clogin.com for b2c authoritie [](start = 2, length = 47)

:)

Reply:
:face_with_rolling_eyes:

---
In reply to: [231366199](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1389#discussion_r231366199) [](ancestors = 231366199)